src/amo/components/AddonMoreInfo/index.js

```diff
@@ -203,6 +203,24 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
           </Link>
         </li>
       ),
+      tagsLinks:
+        addon.tags != null && addon.tags.length > 0
```

Comment:
Were tags already stored in the redux store before? I think we should normalize `tags` so that this prop is an array all the time, this should be done in the addons reducer (in `createInternalAddon()` I'd say)

Reply:
Can the API return duplicate tags?

Reply:
Tags were already there in the redux store.  I read https://github.com/mozilla/addons-frontend/blob/master/src/amo/types/addons.js#L142 as it could be missing/null though (should I change it to just `tags: Array<string>,` ?

Reply:
On the server side, the API always returns a list/array for tags - empty if none are set.  There's a db constraint on `AddonTag` to prevent duplicate tags.

Reply:
> Tags were already there in the redux store. I read [https://github.com/mozilla/addons-frontend/blob/master/src/amo/types/addons.js#L142](https://github.com/mozilla/addons-frontend/blob/master/src/amo/types/addons.js?rgh-link-date=2021-07-28T11%3A37%3A27Z#L142) as it could be missing/null though (should I change it to just `tags: Array<string>,` ?

I think we could mark it as required indeed (= what you suggested)

Reply:
done in https://github.com/mozilla/addons-frontend/pull/10798/commits/cc3e2189f853e163202b0d2f14905f950229474a